gather_stats.js

```diff
@@ -53,6 +50,7 @@ async function retrieve_nodes_from_network() {
                 format: '{bar} {percentage}% | ETA: {eta}s | {value}/{total} {name}',
                 autopadding: true,
                 hideCursor: true,
+                noTTYOutput: process.env.NO_TTY === 'TRUE',

```

Comment:
[progress updates are only visible in TTY mode!](https://github.com/AndiDittrich/Node.CLI-Progress/blob/master/lib/multi-bar.js#L36) . There would be a null-reference exception if `noTTYOutput` option is not set